sdk/template/template/review/template.api.md

```diff
@@ -1,44 +0,0 @@
-## API Report File for "@azure/template"
```

Comment:
So, this file should still exist, there just shouldn't be any changes to it as a result of our PR. :smile:

Reply:
This still needs to be addressed.